### PR TITLE
Feat: Add Rider Model

### PR DESCRIPTION
**Pull request by OmarMasoud3**

### CODE REVIEW

File: server/server/models/rider.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        required: true
+    },
+    license: {
+        type: String,
+        required: true,
+        unique: true
+    },
+    nationalityId: {
+        type: String,
+        required: true
```
Comment (Dana-MO) [inline server/server/models/rider.js:16]: could we add unique property to nationalityId

**Review by Dana-MO — CHANGES_REQUESTED** [review]
Great job, just a small request to add.

**Review by MahmoudRumaneh — APPROVED** [review]
Great job Omar, everything is good just the comment that Dana mentioned

File: server/server/models/rider.js (hunk 1 above)
Comment (OmarMasoud3) [reply server/server/models/rider.js:16]: @Dana-MO definitely, on it

**Review by Dana-MO — APPROVED** [review]

**Pull request merged**